src/blocking.rs

```diff
@@ -50,6 +50,13 @@ impl BlockingIronOxide {
         &self.ironoxide.device
     }
 
+    /// Clears all entries from the policy cache.
+    /// # Returns
+    /// Number of entries cleared from the cache
```

Comment:
Maybe we should use a See comment instead. ` /// See [ironoxide::document::DocumentOps::document_list()](trait.DocumentOps.html#tymethod.document_list)` for instance

Reply:
Done